### DISCUSSION TITLE
Ruby 3.2 - Needs unreleased fixes

### DISCUSSION THREAD

**Discussion by davidwessman** [shrinerb/shrine#641] [General]
When upgrading to Ruby 3.2, we ran into the issue fixed by https://github.com/shrinerb/shrine/commit/004ffc8389aed040ba6b19f78ed937cfafa18dfb.

Would you be able to make a release soon? 🙂

**Comment by janko** [comment]
Yeah, I'm aware of this Ruby 3.2 compatibility issue, I experienced it myself recently when upgrading to Ruby 3.2, but worked around it by using strings instead of booleans (i.e. `remove_attachment = "true"`). I'm getting ready to make a new release soon 🤞🏻
